ding/utils/comm_perf_helper.py

```diff
@@ -0,0 +1,111 @@
+import torch
+import functools
+import time
+from concurrent import futures
+from ditk import logging
+from typing import List, Optional, Tuple, Dict, Any
+
+UNIT_1_B = 1
+UNIT_1_KB = 1024 * UNIT_1_B
+UNIT_1_MB = 1024 * UNIT_1_KB
+UNIT_1_GB = 1024 * UNIT_1_MB
+TENSOR_SIZE_LIST = [
+    8 * UNIT_1_B, 32 * UNIT_1_B, 64 * UNIT_1_B, UNIT_1_KB, 4 * UNIT_1_KB, 64 * UNIT_1_KB, 1 * UNIT_1_MB, 4 * UNIT_1_MB,
+    64 * UNIT_1_MB, 512 * UNIT_1_MB, 1 * UNIT_1_GB, 2 * UNIT_1_GB, 4 * UNIT_1_GB
+]
+# TENSOR_SIZE_LIST = [
+#     8 * UNIT_1_B, 32 * UNIT_1_B, 64 * UNIT_1_B, UNIT_1_KB, 4 * UNIT_1_KB, 64 * UNIT_1_KB, 1 * UNIT_1_MB, 4 * UNIT_1_MB,
+#     64 * UNIT_1_MB, 512 * UNIT_1_MB
+# ]
+
+DO_PERF = False
+TYPE_MAP = {torch.float32: 4, torch.float64: 8, torch.int32: 4, torch.int64: 8}
```

Comment:
we usually only use torch.float32, torch.int64, torch.uint8

Reply:
Done